Library/Homebrew/cmd/audit.rb

```diff
@@ -193,6 +194,9 @@ def audit_file
     if present.include?("head") && present.include?("head block")
       problem "Should not have both `head` and `head do`"
     end
+    if present.include?("bottle modifier") && present.include?("bottle block")
```

Comment:
This is not related to *this* PR per se but we should probably use symbols instead of strings for the components, e.g. `:bottle_modifier`/`:test_block`/etc. It’d speed up the lookup in `present`. Or maybe use a `set`.